src/main/peloton/peloton.cpp

```diff
@@ -49,8 +49,18 @@ void RunPelotonBrain() {
   // TODO(tianyu): register jobs here
   struct timeval one_second;
   one_second.tv_sec = 1;
-  peloton::brain::ExampleBrainJob job;
-  brain.RegisterJob(&one_second, "test", job);
+  one_second.tv_usec = 0;
+
+  auto example_task = [](peloton::brain::BrainEnvironment *) {
+    // TODO(tianyu): Replace with real address
+    capnp::EzRpcClient client("localhost:15445");
+    PelotonService::Client peloton_service = client.getMain<PelotonService>();
+    auto request = peloton_service.createIndexRequest();
+    request.getRequest().setIndexKeys(42);
+    auto response = request.send().wait(client.getWaitScope());
```

Comment:
I think we talked about adding a shim layer to wrap around the RPC implementation, right? Right now the capnp proto stuff is exposed to other parts of the system.